report.js

```diff
@@ -34,7 +34,9 @@ module.exports = {
      * @return {string|null} - return audit result as a json if success, null otherwise
      */
     runAudit: function(root_path){
-        var data = Common.systemSync('ls', root_path);
+        var islinux = process.platform === "linux";
+        var command = islinux ? 'ls' : 'dir';
+        var data  = Common.systemSync(command, root_path);
```

Comment:
This leaves a lot of room for failure (ie if the user has a custom ls or dir command).  I would use `fs.existAsync(root_path)`

https://nodejs.org/api/fs.html#fs_fs_existssync_path